### PR TITLE
STY: Manual conversions to f-strings

### PR DESCRIPTION
**Pull request by DimitriPapadopoulos**
## Changes proposed in this pull request

These have not caught by pyupgrade or ruff.

See https://github.com/nipreps/fmriprep/pull/3147#issuecomment-1971808094.

### CODE REVIEW

File: fmriprep/interfaces/reports.py [hunk 1]
```diff
@@ -169,10 +169,9 @@ def _generate_segment(self):
         if counts:
             header = '\t\t<ul class="elem-desc">'
             footer = '\t\t</ul>'
+            s = '' if n_runs == 1 else 's'
```
Comment (effigies) [inline fmriprep/interfaces/reports.py:172]: `n_runs` isn't defined outside the loop.

```suggestion
```

File: fmriprep/interfaces/reports.py [hunk 2]
```diff
@@ -169,10 +169,9 @@ def _generate_segment(self):
...
             lines = [
-                '\t\t\t<li>Task: {task_id} ({n_runs:d} run{s})</li>'.format(
-                    task_id=task_id, n_runs=n_runs, s='' if n_runs == 1 else 's'
-                )
+                f'\t\t\t<li>Task: {task_id} ({n_runs:d} run{s})</li>'
```
Comment (effigies) [inline fmriprep/interfaces/reports.py:174]: A more terse version of `"s" if n_runs > 1 else ""`:

```suggestion
                f'\t\t\t<li>Task: {task_id} ({n_runs:d} run{"s"[:n_runs > 1]})</li>'
```

File: fmriprep/interfaces/reports.py (hunk 1 above)
Comment (DimitriPapadopoulos) [reply fmriprep/interfaces/reports.py:172]: You mean outside `if`? Yes, it's a local variable.

File: fmriprep/interfaces/reports.py (hunk 2 above)
Comment (DimitriPapadopoulos) [reply fmriprep/interfaces/reports.py:174]: 😀 Nice. But perhaps too terse?

File: fmriprep/interfaces/reports.py (hunk 1 above)
Comment (effigies) [reply fmriprep/interfaces/reports.py:172]: No, it's defined in the comprehension: `lines = [... for task_id, n_runs in sorted(counts.items())]`

Comment (DimitriPapadopoulos) [reply fmriprep/interfaces/reports.py:172]: Oh, of course. My wrong.

File: fmriprep/interfaces/reports.py (hunk 2 above)
Comment (DimitriPapadopoulos) [reply fmriprep/interfaces/reports.py:174]: It still fails. I'll probably revert.

**Review by effigies — APPROVED** [review]

**Review by effigies — APPROVED** [review]

**Pull request merged**